### DISCUSSION TITLE
release date of version 4 stable

### DISCUSSION THREAD

**Discussion by crisfcodes** [react-native-elements/react-native-elements#3361] [Q&A]
is there any document/roadmap for the release of version 4 stable?

**Comment by arpitBhalla** [comment]
Hey, there is some issue with pkg naming on NPM, You can try using Bleeding Edge version (It's quite stable as of now). Try 
```bash
# Install peer dependency base pkg
yarn add react-native-elements/react-native-elements#base

# this will create an alias name for pkg, which might help to migrate to v4
yarn add react-native-elements@react-native-elements/react-native-elements#themed
```
You may need to check [Migration Guide](https://reactnativeelements.com/docs/next/migration_guide#migrating-to-v4)
